langlib/lang.string/src/main/ballerina/string.bal

```diff
@@ -273,6 +385,12 @@ public isolated function fromCodePointInt(int codePoint) returns Char|error = @j
 # Adds sufficient `padChar` characters at the start of `str` to make its length be `len`.
 # If the length of `str` is >= `len`, returns `str`.
 #
+# ```ballerina
+# "100Km".padStart(10, "0") ⇒      100Km
```

Comment:
```suggestion
# "100Km".padStart(10, "0") ⇒ 100Km
```

Reply:
Should be
```suggestion
# "100Km".padStart(10, "0") ⇒ 00000100Km
```

Reply:
Yeah. I just checked it gives `00000100Km`

Reply:
My intension is to cover the default. Corrected the example now.

Reply:
Got it